package.json

```diff
@@ -34,7 +34,7 @@
     "lint:prettier": "prettier --list-different \"**/*.{js,json,ts,tsx,md}\"",
     "lint:types": "tsc",
     "test": "yarn lint:prettier && yarn lint:types",
-    "test:unit": "DATABASE_URL=file:./keystone.db yarn jest",
+    "test:unit": "DATABASE_URL=file:./keystone.db yarn jest -- --runInBand",
```

Comment:
Not strictly a reason for the broken tests, but, technically its a requirement that we rely on the tests running in sequence.